packages/@okta/vuepress-site/docs/guides/social-login/main/index.md

```diff
@@ -86,22 +78,43 @@ An example of a complete URL looks like this:
 https://${yourOktaDomain}/oauth2/v1/authorize?idp=${idp_id}&client_id=${client_id}&response_type=id_token&response_mode=fragment&scope=openid%20email&redirect_uri=https%3A%2F%2FyourAppUrlHere.com%2F&state=WM6D&nonce=YsG76jo
```

Comment:
not part of your change, but is there a way to make this wrap? maybe changing the syntax highlighting from bash? Just noticing that it's sort of scrunched and cut off. Hard to see the copy button.

<img width="832" alt="Screen Shot 2022-06-28 at 3 45 10 PM" src="https://user-images.githubusercontent.com/75638829/176316785-681fa399-0e98-4c50-b95d-ca4d4043351a.png">

Reply:
I don't think I can set a specific code box to wrap. It's a global style for all the code snippets.
However, I can add a newline to push the copy button below the URL text. Does this help?
![URL_code_copy](https://user-images.githubusercontent.com/80703015/176483014-984ef2e8-06da-4675-8867-4662665c42d8.png)

Reply:
ahh ok. Nevermind then. Moving the button down looks a little better for me, but whatever you think is best.